tequilapi/endpoints/identities.go

```diff
@@ -33,6 +44,87 @@ func (endpoint *identitiesApi) List(writer http.ResponseWriter, request *http.Re
 	utils.WriteAsJson(idsSerializable, writer)
 }
 
+func (endpoint *identitiesApi) Create(writer http.ResponseWriter, request *http.Request, _ httprouter.Params) {
+	createReq, err := toCreateRequest(request)
+	if err != nil {
+		utils.SendError(writer, err, http.StatusBadRequest)
+		return
+	}
+	errorMap := validateCreationRequest(createReq)
+	if errorMap.HasErrors() {
+		utils.SendValidationErrorMessage(writer, errorMap)
+		return
+	}
+	id, err := endpoint.idm.CreateNewIdentity(createReq.Password)
+	if err != nil {
+		utils.SendError(writer, err, http.StatusInternalServerError) // This should never happen
+		return
+	}
+
+	idDto := identityDto{string(id)}
+	utils.WriteAsJson(idDto, writer)
+}
+
+func (endpoint *identitiesApi) Register(writer http.ResponseWriter, request *http.Request, params httprouter.Params) {
+	id, registerReq, err := toRegisterRequest(request, params)
+	if err != nil {
+		utils.SendError(writer, err, http.StatusBadRequest)
+		return
+	}
+	err = validateRegistrationRequest(registerReq)
+	if err != nil {
+		utils.SendError(writer, err, 501)
```

Comment:
- is it really validation?
- If so, validation errors should be 400 formatted always with `utils.SendValidationErrorMessage()`